common/src/main/kotlin/org/amshove/kluent/internal/Assertions.kt

```diff
@@ -57,6 +57,12 @@ internal fun failExpectedActual(message: String, expected: String?, actual: Stri
     |   Actual:     $actual
 """.trimMargin())
 
+internal fun failCollectionWithDifferentItems(message: String, expected: String?, actual: String?): Nothing = fail("""
+    |$message
+    |${ if(actual.isNullOrEmpty()) "Items included on the expected collection but not in the actual: $expected" else "" }
```

Comment:
I think this should be *not* `isNullOrEmpty` :)
(same below)

Reply:
My logic was a bit confusing here... If the `actual` variable has items (not empty) it shows what was `expected`. I've changed to use *not* and used the same variable on each line (as you suggested).